llvm/test/Transforms/HipStdPar/math-fixup.ll

```diff
@@ -0,0 +1,243 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -passes=hipstdpar-math-fixup %s | FileCheck %s
+
+define void @foo(double noundef %dbl, float noundef %flt, i32 noundef %quo) #0 {
```

Comment:
All in one test functions are harder to review and merge, can you split this at least into only covering the related functions in one caller

Reply:
Done.